app/uk/gov/hmrc/gform/testonly/TestOnlyController.scala

```diff
@@ -625,10 +674,10 @@ class TestOnlyController(
     val tableRows: List[List[TableRow]] = snapshots.map { snapshot =>
       List(
         TableRow(
-          content = HtmlContent(Html(snapshot.templateId))
+          content = HtmlContent(Html(snapshot.templateId.value))
         ),
         TableRow(
-          content = HtmlContent(Html(snapshot.description))
+          content = HtmlContent(Html(snapshot.description.value))
         ),
         TableRow(
           content = Text(formatInstant(snapshot.savedAt))
```

Comment:
Use `uk.gov.hmrc.gform.views.html.formatInstant` and remove duplicate `formatInstant` this class.